config/Dockerfiles/rpmbuild/rpmbuild-test.sh

```diff
@@ -1,79 +1,138 @@
+     echo "description='${fed_repo} - make test failed'" >> ${LOGDIR}/package_props.txt
+     exit $MAKE_TEST_STATUS
 fi
-# Build the package into ./results_${fed_repo}/$VERSION/$RELEASE/
+# Prepare concurrent koji build
+cp -rp ../${fed_repo} /root/rpmbuild/SOURCES/
+rpmbuild -bs /root/rpmbuild/SOURCES/${fed_repo}.spec
+# Set up koji creds
+#TODO
+# Should be a fedora-packager-setup command and a kinit. Will also probably require some packages like fedora-packager/python-krbV
+# Build the package into ./results_${fed_repo}/$VERSION/$RELEASE/ and concurrently do a koji build
+#{ time fedpkg --release ${fed_branch} mockbuild ; } 2> ${LOGDIR}/mockbuildtime.txt & { time koji build --scratch $RSYNC_BRANCH /root/rpmbuild/SRPMS/${fed_repo}*.src.rpm ; } 2> ${LOGDIR}/kojibuildtime.txt && fg
 fedpkg --release ${fed_branch} mockbuild
 MOCKBUILD_STATUS=$?
-sudo echo "status=$MOCKBUILD_STATUS" >> ${OUTPUTDIR}/logs/package_props.txt
-if [ "$MOCKBUILD_STATUS" != 0 ]; then echo "ERROR: FEDPKG MOCKBUILD\nSTATUS: $MOCKBUILD_STATUS"; exit 1; fi
+echo "status=$MOCKBUILD_STATUS" >> ${LOGDIR}/package_props.txt
+if [ "$MOCKBUILD_STATUS" != 0 ]; then echo -e "ERROR: FEDPKG MOCKBUILD\nSTATUS: $MOCKBUILD_STATUS"; exit 1; fi
 popd
 
 ABIGAIL_BRANCH=$(echo ${fed_branch} | sed 's/./&c/1')
 if [ "${fed_branch}" = "master" ]; then
     ABIGAIL_BRANCH="fc27"
 fi
 # Make repo with the newly created rpm
-rm -rf ${OUTPUTDIR}/${fed_repo}_repo
-mkdir ${OUTPUTDIR}/${fed_repo}_repo
-cp /${fed_repo}/results_${fed_repo}/${VERSION}/*/*.rpm ${OUTPUTDIR}/${fed_repo}_repo/
+rm -rf ${RPMDIR}/*
+mkdir -p ${RPMDIR}
+cp /${fed_repo}/results_${fed_repo}/${VERSION}/*/*.rpm ${RPMDIR}/
 # Run rpmlint
-rpmlint ${OUTPUTDIR}/${fed_repo}_repo/ > ${OUTPUTDIR}/logs/rpmlint_out.txt
-pushd ${OUTPUTDIR}/${fed_repo}_repo && createrepo .
+rpmlint ${RPMDIR}/ > ${LOGDIR}/rpmlint_out.txt
+pushd ${RPMDIR} && createrepo .
 popd
 # Run fedabipkgdiff against the newly created rpm
 rm -rf libabigail
-git clone git://sourceware.org/git/libabigail.git
+git clone -q git://sourceware.org/git/libabigail.git
 RPM_TO_CHECK=$(find /${fed_repo}/results_${fed_repo}/${VERSION}/*/ -name "${fed_repo}-${VERSION}*" | grep -v src)
-libabigail/tools/fedabipkgdiff --from ${ABIGAIL_BRANCH} ${RPM_TO_CHECK} &> ${OUTPUTDIR}/logs/fedabipkgdiff_out.txt
+libabigail/tools/fedabipkgdiff --from ${ABIGAIL_BRANCH} ${RPM_TO_CHECK} &> ${LOGDIR}/fedabipkgdiff_out.txt
 RPM_NAME=$(basename $RPM_TO_CHECK)
-echo "package_url=${HTTP_BASE}/${fed_branch}/repo/${fed_repo}_repo/$RPM_NAME" >> ${OUTPUTDIR}/logs/package_props.txt
+echo "package_url=${HTTP_BASE}/${fed_branch}/repo/${fed_repo}_repo/$RPM_NAME" >> ${LOGDIR}/package_props.txt
 echo "original_spec_nvr=${truenvr}" >> ${OUTPUTDIR}/logs/package_props.txt
 RPM_NAME=$(echo $RPM_NAME | rev | cut -d '.' -f 2- | rev)
 echo "nvr=${RPM_NAME}" >> ${OUTPUTDIR}/logs/package_props.txt
+RSYNC_LOCATION="${RSYNC_HOST}::${RSYNC_DIR}/${RSYNC_BRANCH}"
```

Comment:
It just confused me. Which doesn't take much these days. I'll leave it up to you.